e2e/cypress/integration/interactive_dialog/full_dialog_spec.js

```diff
@@ -104,13 +98,7 @@ describe('Interactive Dialog', () => {
                     // * Verify that the suggestion list or autocomplete open up on click of input element
                     cy.wrap($elForm).find('#suggestionList').should('not.exist');
                     cy.wrap($elForm).find('input').click();
-                    cy.wrap($elForm).find('#suggestionList').scrollIntoView().should('be.visible').children().then((el) => {
-                        if (element.name === 'someuserselector' && config.ElasticsearchSettings && config.ElasticsearchSettings.EnableIndexing) {
-                            return;
-                        }
-
-                        cy.wrap(el).should('have.length', optionsLength[element.name]);
-                    });
```

Comment:
Removed this conditional check since this is just a nice to have test and frequently causing flaky tests.